tests/unit/test-util/test.adb.js

```diff
@@ -30,6 +30,7 @@ const fakeRDPUnixSocketFile = (
   '/data/data/org.mozilla.firefox/firefox-debugger-socket'
 );
 
+const artifactDir = `web-ext-artifacts-${Date.now()}`;
```

Comment:
Move this into the related test case, it doesn't seem that we need this to be global.